.env.example

```diff
@@ -0,0 +1,17 @@
+# The endpoint of the server's API that the client application will communicate with
+# E.g., http://localhost:5000/posts
+NEXT_PUBLIC_HOSTED_URL="http://localhost:3000"
+
+# The base URL for the Asgardeo root organization's API
+# E.g., https://api.asgardeo.io/t/your-org
+NEXT_PUBLIC_ASGARDEO_BASE_ORGANIZATION_URL="https://api.asgardeo.io/t/guardioinc"
```

Comment:
Let's keep this as <NEXT_PUBLIC_ASGARDEO_BASE_ORGANIZATION_URL>

Reply:
Fixed